src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -607,6 +607,57 @@ public void setRequestInterceptor(RequestInterceptor interceptor) {
     }
 
     /**
+     * Get a lower-scoped token restricted to a resource for the list of scopes that are passed.
+     * @param scopes the list of scopes to which the new token should be restricted for
+     * @param resource the resource for which the new token has to be obtained
+     * @return scopedToken which has access token and other details
+     */
+    public ScopedToken getLowerScopedToken(List<String> scopes, String resource) {
+        assert (scopes != null);
+        assert (scopes.size() > 0);
+        assert (resource != null);
+        URL url = null;
+        try {
+            url = new URL(this.getTokenURL());
+        } catch (MalformedURLException e) {
+            assert false : "An invalid refresh URL indicates a bug in the SDK.";
+            throw new RuntimeException("An invalid refresh URL indicates a bug in the SDK.", e);
+        }
+        StringBuilder spaceSeparatedScopes = new StringBuilder();
+
+        for (String scope : scopes) {
+            spaceSeparatedScopes.append(scope);
+            spaceSeparatedScopes.append(" ");
+        }
+        //Delete the extra space at the end
+        spaceSeparatedScopes.deleteCharAt(spaceSeparatedScopes.length() - 1);
+
+        //this.getAccessToken() ensures we have a valid access token
+        String urlParameters = String.format("grant_type=urn:ietf:params:oauth:grant-type:token-exchange"
+                + "&subject_token_type=urn:ietf:params:oauth:token-type:access_token&subject_token=%s"
+                + "&scope=%s&resource=%s",
+            this.getAccessToken(), spaceSeparatedScopes, resource);
+        BoxAPIRequest request = new BoxAPIRequest(this, url, "POST");
+        request.shouldAuthenticate(false);
+        request.setBody(urlParameters);
+
+        String json;
+        try {
+            BoxJSONResponse response = (BoxJSONResponse) request.send();
+            json = response.getJSON();
+        } catch (BoxAPIException e) {
+            this.notifyError(e);
+            throw e;
+        }
+
+        JsonObject jsonObject = JsonObject.readFrom(json);
+        ScopedToken token = new ScopedToken(jsonObject);
+        token.setObtainedAt(System.currentTimeMillis());
```

Comment:
Since we don't have the issued time, sadly, this gap is something we have to deal with